components/forms/signup-form.njk

```diff
@@ -6,7 +6,7 @@
   action="https://r1-t.trackedlink.net/signup.ashx"
```

Comment:
Is this link correct?

Reply:
yes, it's the HTTPS link – see yellow info box in https://support.dotmailer.com/hc/en-gb/articles/212211788-Creating-a-signup-form